modules/core/04-channel/keeper/upgrade.go

```diff
@@ -64,43 +66,165 @@ func (k Keeper) WriteUpgradeInitChannel(ctx sdk.Context, portID, channelID strin
 	emitChannelUpgradeInitEvent(ctx, portID, channelID, currentChannel, upgrade)
 }
 
-// upgradeTry
+// ChanUpgradeTry is called by a module to accept the first step of a channel upgrade handshake initiated by
+// a module on another chain. If this function is successful, the proposed upgrade will be returned. If the upgrade fails, the upgrade sequence will still be incremented but an error will be returned.
 func (k Keeper) ChanUpgradeTry(
 	ctx sdk.Context,
 	portID,
 	channelID string,
 	proposedConnectionHops []string,
-	upgradeTimeout types.Timeout,
+	proposedUpgradeTimeout types.Timeout,
 	counterpartyProposedUpgrade types.Upgrade,
 	counterpartyUpgradeSequence uint64,
 	proofCounterpartyChannel,
```

Comment:
for the actual proofs I find the name `counterpartyChannelProof` much better to read more naturally and to avoid confusion with `proofHeight` .

Reply:
i spoke to @colin-axner about this, i think the issue is that this is the convention we currently use all over the rest of the codebase. i'm happy to open up an issue though to rename these params!